app/models/concerns/scholars_archive/article_metadata.rb

```diff
@@ -32,6 +33,50 @@ module ArticleMetadata
       property :web_of_science_uid, predicate: ::RDF::URI.new('http://opaquenamespace.org/ns/webOfScienceUid'), multiple: false do |index|
         index.as :stored_searchable
       end
+
+      define_singleton_method :article_properties do
```

Comment:
Do you have plans for these methods? They don't seem used in this PR and didn't exist before as far as I can tell.

Reply:
Yeah, I wanna get these initial refactors in and get it all playing nicely. This method would replace the model::TERMS which should lower the amount of lists we carry around smaller.